pkg/controller/daemon/config/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
```

Comment:
is this the same list as before? Can we identify the main developers of each controller via `git blame`?